### PR TITLE
some simple refactoring

### PR DESCRIPTION
**Pull request by leeoocca**
Hey! I did some refactoring after finding a couple of typos. I think I may have gone a bit too far 😅 

`index.md`
- fix GitHub spelling

`libs/minilibx/*.md`
- fix typos
- rephrased some sentences
- fix mlx_xpm_file_to_image
- tables instead of lists where applicable
- tabs instead of spaces in code blocks
- consistency with acronyms and punctuation
- some other small styling fixes
- _also_ fix macOS spelling

Tell me what you think!

### CODE REVIEW

File: libs/minilibx/images.md [hunk 1]
```diff
@@ -21,31 +21,29 @@ nav_order: 7
...
+object. This is very useful for textures or sprites of course.
 
 ## Reading images
 
 To read from a file to a image object, you need either the XMP or PNG format. In
-order to read we can call the according functions `mlx_xpm_file_to_image` and
-`mlx_png_file_to_image`. Do mind that `mlx_png_file_to_image` currently leaks
-memory. Both functions accept exactly the same parameters and their usage is
-identical. 
+order to read we can call the functions `mlx_xpm_file_to_image` and `mlx_png_file_to_image` accordingly. Do mind that `mlx_png_file_to_image` currently leaks memory.
```
Comment (harm-smits) [inline libs/minilibx/images.md]: Please adhere to the 80 col line limit.

**Review by harm-smits — CHANGES_REQUESTED** [review]
Thanks for the contribution. I sincerely appreciate it. There are a few things left to do, namely:

1. I used spaces because, those are actually universal. I see you assumed tab size is `4`, whilst by default, its actually `8`. So you will have to create the file `_sass/custom/custom.scss` with the according contents:
```scss
pre {
    tab-size: 4;
    -moz-tab-size: 4;
}
```

2. You should also add a `.editorconfig` file to make sure all the tab characters are rendered to 4 spaces, something like this will do the trick:
```editorconfig
[*.{md,scss}]
charset = utf-8
indent_style = tab
indent_size = 4
```

3. Adhere to the 80 column line limit as much as possible. There is one case where you removed a newline.

Besides that, everything is looking great!

Thanks in advance!

**Comment by leeoocca** [comment]
1. You're right, I assumed tab size was 4. I had already seen tabs here and there so I thought that would be fine. Turns out it isn't. I'm going to add the lines you suggested.
2. Yeah, makes sense. I'll add it right away.
3. Yes, my mistake!

Thank you for your work!

File: libs/minilibx/images.md (hunk 1 above)
Comment (leeoocca) [reply libs/minilibx/images.md]: Done!

**Comment by harm-smits** [comment]
Thanks for the final changes! Merged.

**Pull request merged**